.github/workflows/build.yml

```
@@ -11,17 +11,30 @@ concurrency:
 
 env:
   BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
-  BRANCH_NAME_FORMATTED_FOR_REPORTS: ${BRANCH_NAME//\//_}
   UNIT_TESTS_COVERAGE_REPORTS_PATH: tests/unit/coverage
+  UNIT_TESTS_COVERAGE_BASE_CACHE_KEY: tests-unit-coverage-${{ needs.init.outputs.safe_branch_name }}
```

actionlint (1.7.7)

context "needs" is not allowed here. available contexts are "github", "inputs", "secrets", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)